command/connect/envoy/envoy.go

```diff
@@ -196,13 +195,16 @@ func (c *cmd) Run(args []string) int {
 	}
 	passThroughArgs := c.flags.Args()
 
+	// TODO(mike): uncomment after restoring env var functionality
```

Comment:
TODO per @rboyer's suggestion:
> Maybe those constant env var names should just be moved to some place like `command/connect/env.go`

Reply:
Extracted from ` agent/proxyprocess/proxy.go` and attempted to reimplement in `command/flags/connect.go` following the patterns in `command/flags/http.go`, _definitely_ needs review.

Reply:
Scrapping this approach in favor of just hardcoding the env var strings instead.